### ISSUE TITLE
[Desktop] [Request] Please support opera for "import bookmarks and settings"

### ISSUE THREAD

**Issue by pitsi** [brave/brave-browser#9379]
## Description 
As the title suggests, I would like brave to support opera as one of the browsers in the "import bookmarks and settings" menu. It will make my transition away from it a lot easier.

## Steps to Reproduce
   1. Click the 3-lines menu on the top righ corner.
   2. Go to bookmakrs > import bookmarks and settings
   3. Notice the drop down menu that lists the installed browsers, offering the ability to import bookmarks, history and settings from them.

## Actual result:
Opera is not listed under there, although it is installed.

## Expected result:
Opera should appear in there.

## Reproduces how often: 
Everytime.

## Brave version (brave://version info)
Brave	1.7.98 Chromium: 81.0.4044.113 (Official Build) (64-bit)
Revision	e3225dafb0475864a1812a374d73a92e391635ac-refs/branch-heads/4044@{# 936}
OS	Linux

## Version/Channel Information:
- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? No idea, probably yes,
- Can you reproduce this issue with the dev channel? No idea, probably yes,
- Can you reproduce this issue with the nightly channel? No idea, probably yes,

## Other Additional Information:
- Does the issue resolve itself when disabling Brave Shields? No 
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

## Miscellaneous Information:
Opera's data is stored in ~/.config/opera (and ~/.config/opera-beta and ~/.config/opera-developer for the other channels) and opera, unlike chromium. chrome and firefox does NOT use profiles. There is only one profile and its contents are all in the forementioned folder, so I assume it would be pretty trivial to implement.

**Comment by rebron** [comment]
cc: @simonhong Thoughts? Probably could pull in Vivaldi as well.

**Comment by pitsi** [comment]
I must say that I have completely moved to brave since I opened that issue ~1 month ago, so I won't be able to test it, if it ever becames available. Sorry :(

**Comment by bsclifton** [comment]
@pitsi no worries - welcome to Brave! 😄👍

**Comment by simonhong** [comment]
@rebron Yeah, having option for opera seems nice. I can take this. :)

**Comment by brij1234** [comment]
@simonhong Any updates? I'd really like to move away from "free" Opera browser and start using Brave.

**Comment by pitsi** [comment]
@brij1234 
As a former opera and now brave user, I can give you some hints if you want.
If your concern is just the bookmarks, you can easily export them from opera and import them to brave.

**Comment by mccarthy1993** [comment]
> @brij1234
> As a former opera and now brave user, I can give you some hints if you want.
> If your concern is just the bookmarks, you can easily export them from opera and import them to brave.

Yes, please! The only reason I still have opera installed is my immense and painstakingly organised library of bookmarks that I haven't been able to import.

**Comment by pitsi** [comment]
Can't you just export them from opera and import them to brave? That's what I did 1+ year ago.

**Comment by mccarthy1993** [comment]
> Can't you just export them from opera and import them to brave? That's what I did 1+ year ago.

Good idea. I've now tried exporting all my bookmarks from Opera (it exports as one .html). But there's still the issue of getting all the sites organised (my bookmarks library had folders within folders within folders...) 

Do you know if Brave has a function to effectively (folders included) import bookmarks just from an html file?

**Comment by pitsi** [comment]
For me, the only annoyance was their order. After the import, opera's first (oldest) bookmark was brave's last (newest), so any other bookmark I added afterwards was added after that... 5+ year old bookmark I had from opera

**Comment by mccarthy1993** [comment]
> For me, the only annoyance was their order. After the import, opera's first (oldest) bookmark was brave's last (newest), so any other bookmark I added afterwards was added after that... 5+ year old bookmark I had from opera

Seconded. 

But re-ordering them is far more manageable than re-adding each bookmark manually, so I'm certainly better off than before 😁

**Comment by pitsi** [comment]
So, I noticed this bit in today's changelog for 1.43.x
`Added the ability to import from Opera. (#23738)`

If anyone has opera installed and can verify that it does work, this issue can be closed.

**Comment by pitsi** [comment]
I installed all versions of opera (stable, beta and developer) and I tested it myself.
Brave does import settings from opera, but there is a small catch.

As it seems, brave only searches for `~/.config/opera` for opera's stuff, which is for stable, while beta and developer store their stuff on `~/.config/opera-beta` and `~/.config/opera-developer` respectively. 

p.s. This refers to linux obviously, I have no idea what happens on other operating systems.

**Comment by bsclifton** [comment]
Closing as stale - this should have been done in https://github.com/brave/brave-browser/issues/23738

**Comment by pitsi** [comment]
It is obviously stale, but can brave now import from opera beta and opera developer? 
I do not have either of the two to test it and I do not want to waste my time again.
